### PR TITLE
[NeoDays] Item cleaning, patches, tools, and other misc. additions and changes

### PR DESCRIPTION
**Pull request by Golfavel**
#### Summary
NeoDays "Adds and adjusts some random items from groups, and organizes some random item groups"

#### Content of the change
Adds:
- Hair dryer
- Bath toy
- Professional plunger
- Rolling pin
- Nylon
- Lycra patch
- Nomex
- Cotton patch
- Neoprene
- (default) chestwrap
- Leather chestwrap
- Wool chestwrap
- Heavy duty flashlight (on)
- Gasoline lantern (off)
- Oil lamp (off)
- Clay oil lamp (off + on) (animated when on)

Changes:
- Bark (also adds alder bark)
- Various patches
- Reading lights (atomic included) to separate from baseball

Trims:
- Various armwear
- Scythe
- Glass pipe
- Snare trigger
- Tuba
- Leather Apron

#### Testing
Before:
befmiscsprites
After:
aftmiscsprites
Various other items were checked to assure they still display as before.

#### Additional information
The atomic light and atomic lamp both have a json file due to an issue (intentional?) with vehicle parts that use their sprites not allowing the world items to be assigned the sprites

### CODE REVIEW

File: gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_lamp.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "atomic_lamp"
+  ],
+  "fg": "atomic_lamp",
+  "//": "due to a vehicle part json referring to this sprite, this file is necessary for the item to display"
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_lamp.json:6]: maybe add what part in the comment

File: gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_light.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "atomic_light"
+  ],
+  "fg": "atomic_light",
+  "//": "due to a vehicle part json referring to this sprite, this file is necessary for the item to display"
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_light.json:6]: same say say what part, or combine the two json files in one by putting the item and the vehicle part in the id field

File: gfx/NeoDays/pngs_tiles_10x10/items/rag_tinder.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "rag",
+    "tinder"
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/items/rag_tinder.json:4]: rag and tinder using the same sprite is a bit weird, no?

Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/items/rag_tinder.json:4]: It's how it has been, and I don't have an idea as to what to change the sprite into currently.

File: gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_light.json (hunk 2 above)
Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_light.json]: Done; mentioned the vehicle part file.

File: gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_lamp.json (hunk 1 above)
Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_lamp.json]: Done; mentioned the vehicle part file.

**Pull request merged**